### ISSUE TITLE
[MENU SUGGESTION] Better selection & size

### ISSUE THREAD

**Issue by Persvicx** [iiDk-the-actual/FortniteEmoteWheel#6]
**My issue**
You have to hold down the stick to use the menu. likewise, its hard to select things on some vr controllers. You should make it where you click it once to bring it up, move the joystick to what you want to select, and then click down.

**Other inconveniences**
The menu is WAY too big, you should make it reasonably smaller, probably about 2x the gorilla hand size. its hard to see what you are selecting sometimes. 

**Thank you for making such a great mod, by the way**

**Comment by NitroStorm07** [comment]
Yeah, the menu should work on a toggle system to be honest, its really hard to select a emote at the moment

**Comment by Zyverianox** [comment]
Also they need to  have a button called remove shaking when emoting 🙏
